api/python/provisioner/salt_master.py

```diff
@@ -58,7 +59,7 @@ def check_salt_master_is_restarted(pid_before):
             )
             return False
         else:
-            raise
+            raise SaltMasterError('check_salt_master_is_restarted',exc)
```

Comment:
here and in similar places: the exception includes the information about function, we don't need that redundancy